content/docs/2.4/operate/metrics-server.md

```diff
@@ -74,4 +74,24 @@ During its work, KEDA updates each ScaledObject with some relevant information w
 You can recover the metric names from a ScaledObject using `kubectl`:
 ```bash
  kubectl get scaledobject SCALEDOBJECT_NAME -n NAMESPACE -o jsonpath={.status.externalMetricNames}
+```
+
+## How to get metric if there are more than 1 ScaledObject with same metric name
```

Comment:
```suggestion
## How to get metric when multiple ScaledObjects have the same metric name
```

Reply:
> LGTM, but added some suggestions for all versions

Nice, I will update all of them, thanks for the text improvements :)